ui/src/components/easter/invitation/InvitationContainer.js

```diff
@@ -1,11 +1,163 @@
-import { Text } from "@chakra-ui/react";
+import { 
+  Button, 
+  Heading, 
+  Text, 
+  HStack, 
+  Container, 
+  useMediaQuery,
+  UnorderedList,
+  OrderedList,
+  ListItem,
+  Link } from "@chakra-ui/react";
+import { generateGoogleCalendarLink } from "../../helpers/eventsHelpers";
+import { domToReact, attributesToProps } from "html-react-parser";
+
+function Event_details({header , sub_header , sentence , date , time , location , eventData}) {
+  let hColor , backgroundColor , borderColor , hoverColor;
+  if(header == "Good Friday"){
+    [hColor , backgroundColor , borderColor, hoverColor] = ["#A6657A" , "#F1D8DE" , "#C69494" , "#D28F8F"] ; 
+  }
+  else{
+    [hColor , backgroundColor , borderColor, hoverColor] = ["#436EA0" , "#E0EDFF" , "#6E7F98" , "#7F99C0"]
+  } 
+  
+  const options = {
```

Comment:
is options ever used in this file? i'm guessing most of these code you copied from EventCards (which is good lol), but i don't think options here is used at all